include/votca/xtp/segment.h

```diff
@@ -0,0 +1,196 @@
+    void             setOcc(double occ, int e_h_s_t);
+
+    // state: -1 electron +1 hole +2 singlet +3 triplet
+  
+    /// Following notation can be observed in: 
+    /// [1. Victor, R. et al. Microscopic Simulations of Charge Transport in Disordered Organic Semiconductors. J. Chem. Theory Comput. 7, 3335–3345 (2011).] 
+    /// Labeling of the following methods follows the following semantics:
+    /// U - Energy 
+    /// n - neutral geometry
+    /// N - neutral state
+    /// c - charged geometry
+    /// C - charged state
+    /// x - excited geometry
+    /// X - excited state
+
+    /// UcC - UnN
+    void             setU_cC_nN(double dU, int state);
+    /// UnN - UnN
+    void             setU_nC_nN(double dU, int state);
+    /// UcN - UcC
+    void             setU_cN_cC(double dU, int state);
+    /// UxX - UnN
+    void             setU_xX_nN(double dU, int state);
+    /// UnX - UnN
+    void             setU_nX_nN(double dU, int state);
+    /// UxN - UxX
+    void             setU_xN_xX(double dU, int state);
+    const double    &getU_cC_nN(int state);
+    const double    &getU_nC_nN(int state);
+    const double    &getU_cN_cC(int state);
+    const double    &getU_xX_nN(int state);
+    const double    &getU_nX_nN(int state);
+    const double    &getU_xN_xX(int state);
+    double           getSiteEnergy(int state);
+
+    double           getEMpoles(int state);
+    void             setEMpoles(int state, double energy);
+    bool             hasChrgState(int state) { return _hasChrgState[state+1]; }
+    void             setChrgStates(std::vector<bool> yesno) { _hasChrgState = yesno;}
+
+    inline void      setTopology(Topology *container) { _top = container; }
+    Topology        *getTopology() { return _top; }
+    inline void      setMolecule(Molecule *container) { _mol = container; }
+    Molecule        *getMolecule() { return _mol; }
+    inline void      setType(SegmentType *type) { _typ = type; }
+    SegmentType     *getType() { return _typ; }
+
+    void             AddFragment( Fragment* fragment );
+    void             AddAtom( Atom* atom );
+    void             AddPolarSite(PolarSite *pole);
+    void             AddAPolarSite(APolarSite *pole);
+    std::vector< Fragment* > &Fragments() { return _fragments; }
+    std::vector < Atom* >    &Atoms() { return _atoms; }
+    std::vector<PolarSite*>  &PolarSites() { return _polarSites; }
+    std::vector<APolarSite*> &APolarSites() { return _apolarSites; }
+
+
+    void Rigidify();
+
+private:
+
+    int         _id;
+    std::string      _name;
+    SegmentType *_typ;
+    Topology    *_top;
+    Molecule    *_mol;
+
+    std::vector < Fragment* >    _fragments;
+    std::vector < Atom* >        _atoms;
+    std::vector < PolarSite* >   _polarSites;
+    std::vector < APolarSite* >  _apolarSites;
+
+    tools::vec         _CoM;
+    double   _approxsize;
+
+
+    double _U_cC_nN_e;   // from ::EInternal     input     DEFAULT 0
```

Comment:
maybe put all of the organisation energies into a vector later on to make the whole thing more clean